recipes/codecov/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "codecov" %}
+{% set version = "2.0.6" %}
+{% set sha256 = "999bf4d6f7e072433601798f3f291277a4ab519b245af2971cdeb247dbb21454" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  # Not on pypi (at time of packaging)
+  url: https://github.com/codecov/codecov-python/archive/v{{ version }}.tar.gz
```

Comment:
Could you please ask them to upload it to PyPI? Seems strange they haven't done it by now.